### DISCUSSION TITLE
Does MIA Console support artifact registries?

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#271] [Q&A]
### Which section?

Console

### Which is the version?

latest

### Description

Hi

I have a microservice that contains a reference to an internal library, that I want to push to a private artifact registry.
Does the CI/CD tool provided by the Console Installation on Gitlab support the push of such libraries into an artifact registry? 

Thanks

**Comment by fredmaggiowski** [comment]
Hi, this does not depend on the console itself but rather on how you gitlab infrastructure is managed, if your repository is using pipelines templates there are already a set of ready-to-use jobs that you can customize with a few variables, however you are in complete control over your repository pipeline, therefore if you have special needs that the templates do not satisfy you can always write your own from scratch (or compose it by reusing template jobs and overwriting based on your needs).

With that said, if I got your problem correctly you want to push on a different registry than the docker nexus installation provided by Mia-Platform, am I right?

If that is correct you can:

- change the docker registry variable by overriding the build job and set you custom value for the `NEXUS_URL` variable, or:
- ovveride the job named `.docker-job` and set your custom `REMOTE_IMAGE_NAME` variable (please note that this will risk inconsistencies with the `IMAGE_NAME` variable
